cmake/GoogleCloudCppLibrary.cmake

```diff
@@ -349,7 +318,40 @@ function (google_cloud_cpp_add_gapic_library library display_name)
         DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${library_target}"
         COMPONENT google_cloud_cpp_development)
 
-    google_cloud_cpp_install_mocks("${library}" "${display_name}")
+    if (GOOGLE_CLOUD_CPP_WITH_MOCKS)
+        # Create a header-only library for the mocks. We use a CMake `INTERFACE`
+        # library for these, a regular library would not work on macOS (where
+        # the library needs at least one .o file). Unfortunately INTERFACE
```

Comment:
No action needed: I wonder if this is still true now that we use CMake >= 3.13 ..

Reply:
Ack, I don't know the answer and I don't have easy access to a mac.